src/main/java/com/jvavateam/carsharingapp/mapper/RentalMapper.java

```diff
@@ -0,0 +1,28 @@
+package com.jvavateam.carsharingapp.mapper;
+
+import com.jvavateam.carsharingapp.config.MapperConfiguration;
+import com.jvavateam.carsharingapp.dto.rental.CreateRentalDto;
+import com.jvavateam.carsharingapp.dto.rental.CreateRentalResponseDto;
+import com.jvavateam.carsharingapp.dto.rental.RentalReturnResponseDto;
+import com.jvavateam.carsharingapp.model.Rental;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfiguration.class)
+public interface RentalMapper {
+    @Mapping(target = "carId", source = "car.id")
+    @Mapping(target = "userId", source = "user.id")
+    CreateRentalResponseDto toCreateDto(Rental rental);
+
+    @Mapping(target = "carId", source = "car.id")
+    @Mapping(target = "userId", source = "user.id")
+    RentalReturnResponseDto toReturnDto(Rental rental);
```

Comment:
It's better to ignore rental id in response DTO

Reply:
No, I disagree we need ID in response